cmd/add_container.go

```diff
@@ -234,3 +277,77 @@ func generatePipelineDirFiles(dir, workflow, action, pipelineName string) error
 
 	return nil
 }
+
+func filesGeneratedWithSplit(dir string) bool {
+	if _, err := os.Stat(dir + "/api.yaml"); errors.Is(err, os.ErrNotExist) {
+		return false
+	}
+
+	if _, err := os.Stat(dir + "/dependencies.yaml"); errors.Is(err, os.ErrNotExist) {
+		return false
+	}
+
+	return true
+}
+
+func workflowFileFound(workflowDir string) bool {
+	if _, err := os.Stat(workflowDir + "workflow.yaml"); errors.Is(err, os.ErrNotExist) {
+		return false
+	}
+	return true
```

Comment:
Stat can:
- error because `ErrNotExist`
- error for other reasons (no permissions to read dir)
- not error

i dont think the middle scenario is getting caught atm.

Reply:
is this a duplication of `func fileExists(filePath string) bool` below?

Reply:
I think I got rid of that function in a later commit

Reply:
We also left this as is because if there are any other errors they'll be caught when we try to read the file, which returns the error (and all we care about here is whether the file exists)